mongodb/intro/date-aggregation/aggregation-with-date-expression-operators-ii.md

```diff
@@ -0,0 +1,162 @@
+---
+author: Stefan-Stojanovic
+
+aspects:
+  - introduction
+  - new
```

Comment:
we need to move the `aspects` on the workout level (i.e. remove the `aspects` field from all the insights and add it in the workout README)

also, I think we should use different aspects. I think `workout` and `deep` are good for this workout